cmd/juju/commands/bootstrap_test.go

```diff
@@ -1804,33 +1810,32 @@ func (s *BootstrapSuite) TestBootstrapPrintClouds(c *gc.C) {
 
 	ctx, err := cmdtesting.RunCommand(c, s.newBootstrapCommand(), "--clouds")
 	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(cmdtesting.Stdout(ctx), jc.DeepEquals, `
-You can bootstrap on these clouds. See ‘--regions <cloud>’ for all regions.
-Cloud                            Credentials  Default Region
-aws                              fred         us-west-1
-                                 mary         
-aws-china                                     
-aws-gov                                       
-azure                                         
-azure-china                                   
-cloudsigma                                    
-google                                        
-joyent                                        
-oracle                                        
-oracle-classic                                
-rackspace                                     
-localhost                                     
-dummy-cloud                      joe          home
-dummy-cloud-dummy-region-config               
-dummy-cloud-with-config                       
-dummy-cloud-with-region-config                
-dummy-cloud-without-regions                   
-many-credentials-no-auth-types                
-
-You will need to have a credential if you want to bootstrap on a cloud, see
-‘juju autoload-credentials’ and ‘juju add-credential’. The first credential
-listed is the default. Add more clouds with ‘juju add-cloud’.
-`[1:])
+	expectedOut := "You can bootstrap on these clouds. See ‘--regions <cloud>’ for all regions.\n" +
```

Comment:
Why the change from `` to string concatenation? The original way is better and easier to edit

Reply:
the original one has lots of trailing spaces on most of lines.
tests would be failed due to those spaces are automatically removed by linter..
it's quit annoying that I have to disable linter whenever i have to `ctrl+s` the file.